Content.Client/Options/UI/Tabs/MiscTab.xaml

```diff
@@ -23,6 +23,15 @@
                    StyleClasses="LabelKeyText"/>
             <CheckBox Name="ReducedMotionCheckBox" Text="{Loc 'ui-options-reduced-motion'}" />
             <CheckBox Name="EnableColorNameCheckBox" Text="{Loc 'ui-options-enable-color-name'}" />
+            <BoxContainer Orientation="Horizontal">
+            <Label Text="{Loc 'ui-options-chat-window-opacity'}" Margin="8 0" />
+                <Slider Name="ChatWindowOpacitySlider"
+                        MinValue="0"
+                        MaxValue="100"
+                        Rounded="True"
```

Comment:
Can you remove rounding and just make the slider 0.0f to 1.0f? The backend looks like it's 0.0-1.0f and doesn't seem to need 0-100 other than rounding, which isn't necessary either.